src/PhpSpec/Factory/ReflectionFactory.php

```diff
@@ -16,6 +16,8 @@
 /**
  * Class ReflectionFactory is a simple factory wrapper to create reflection
  * classes
+ *
+ * @deprecated
```

Comment:
Why is this deprecated? What else should I use instead? When will it be removed?

Reply:
It was not used for anything, not even the tests

Reply:
I mean, you should just answer these questions in the `@deprecated` comment I think. That's how deprecation comments are done in Symfony and that's quite helpful.

Reply:
I guess I should start triggering warnings?

Reply:
As well, yep!